src/lib/components/SgtDataTable/SgtDataTable.vue

```diff
@@ -422,13 +407,16 @@ export default class SgtDataTable extends Vue {
   }
 }
 </script>
-<style lang="scss" scoped>
```

Comment:
Added a class "sgt-data-table" to the outermost component and nested all the styles within this class selector to avoid any overlap with another component.